framework/encode/capture_manager.cpp

```diff
@@ -610,6 +611,12 @@ void CaptureManager::CheckContinueCaptureForWriteMode()
                 capture_mode_ = kModeDisabled;
                 DestroyStateTracker();
                 compressor_ = nullptr;
+
+                if (quit_after_frame_ranges_)
+                {
+                    GFXRECON_LOG_INFO("All trim ranges have been captured. Quitting.");
+                    exit(0);
```

Comment:
This skips `DestroyInstance` which calls `~CaptureManager()` which calls `util::PageGuardManager::Destroy();`.  I think exiting is relatively safe or we'd have real problems when applications crash and cause GFXR to terminate unexpectedly.  Still, what do you think about a more formal shutdown path here which closes out CaptureManager, calling the virtual dtor for the derived capture managers?

Reply:
@panos-lunarg

Reply:
`exit(0)` still exists but now I am installing an `atexit` function which should be called right after `exit()` and call the destructor